Traceback (most recent call last):
  File "/gpfs/home5/scur2687/erwin/experiments/train_shapenet.py", line 146, in <module>
    fit(config, model, optimizer, scheduler, train_loader, valid_loader, test_loader, 110, 160)
  File "/gpfs/home5/scur2687/erwin/experiments/../../erwin/training.py", line 194, in fit
    benchmark_flops(model, train_loader, config)
  File "/gpfs/home5/scur2687/erwin/experiments/../../erwin/training.py", line 155, in benchmark_flops
    res = model.training_step(sample_batch)["train/loss"]
  File "/gpfs/home5/scur2687/erwin/experiments/../../erwin/experiments/wrappers/shapenet.py", line 46, in training_step
    return self.step(batch, "train")
  File "/gpfs/home5/scur2687/erwin/experiments/../../erwin/experiments/wrappers/shapenet.py", line 40, in step
    pred = self(**batch).squeeze(-1)
  File "/gpfs/home5/scur2687/erwin/erwin/lib64/python3.9/site-packages/torch/nn/modules/module.py", line 1736, in _wrapped_call_impl
    return self._call_impl(*args, **kwargs)
  File "/gpfs/home5/scur2687/erwin/erwin/lib64/python3.9/site-packages/torch/nn/modules/module.py", line 1844, in _call_impl
    return inner()
  File "/gpfs/home5/scur2687/erwin/erwin/lib64/python3.9/site-packages/torch/nn/modules/module.py", line 1790, in inner
    result = forward_call(*args, **kwargs)
  File "/gpfs/home5/scur2687/erwin/experiments/../../erwin/experiments/wrappers/shapenet.py", line 30, in forward
    node_features = self.pos_enc(node_positions)
  File "/gpfs/home5/scur2687/erwin/erwin/lib64/python3.9/site-packages/torch/nn/modules/module.py", line 1736, in _wrapped_call_impl
    return self._call_impl(*args, **kwargs)
  File "/gpfs/home5/scur2687/erwin/erwin/lib64/python3.9/site-packages/torch/nn/modules/module.py", line 1844, in _call_impl
    return inner()
  File "/gpfs/home5/scur2687/erwin/erwin/lib64/python3.9/site-packages/torch/nn/modules/module.py", line 1790, in inner
    result = forward_call(*args, **kwargs)
  File "/gpfs/home5/scur2687/erwin/experiments/../../erwin/experiments/wrappers/shapenet.py", line 12, in forward
    proj = self.linear(x)
  File "/gpfs/home5/scur2687/erwin/erwin/lib64/python3.9/site-packages/torch/nn/modules/module.py", line 1736, in _wrapped_call_impl
    return self._call_impl(*args, **kwargs)
  File "/gpfs/home5/scur2687/erwin/erwin/lib64/python3.9/site-packages/torch/nn/modules/module.py", line 1844, in _call_impl
    return inner()
  File "/gpfs/home5/scur2687/erwin/erwin/lib64/python3.9/site-packages/torch/nn/modules/module.py", line 1790, in inner
    result = forward_call(*args, **kwargs)
  File "/gpfs/home5/scur2687/erwin/erwin/lib64/python3.9/site-packages/torch/nn/modules/linear.py", line 125, in forward
    return F.linear(input, self.weight, self.bias)
  File "/gpfs/home5/scur2687/erwin/erwin/lib64/python3.9/site-packages/torchtnt/utils/flops.py", line 231, in __torch_dispatch__
    rs = func(*args, **kwargs)
  File "/gpfs/home5/scur2687/erwin/erwin/lib64/python3.9/site-packages/torch/_ops.py", line 716, in __call__
    return self._op(*args, **kwargs)
RuntimeError: Expected all tensors to be on the same device, but found at least two devices, cpu and cuda:0! (when checking argument for argument mat2 in method wrapper_CUDA_mm)
Traceback (most recent call last):
  File "/gpfs/home5/scur2687/erwin/experiments/train_shapenet.py", line 146, in <module>
    fit(config, model, optimizer, scheduler, train_loader, valid_loader, test_loader, 110, 160)
  File "/gpfs/home5/scur2687/erwin/experiments/../../erwin/training.py", line 194, in fit
    benchmark_flops(model, train_loader, config)
  File "/gpfs/home5/scur2687/erwin/experiments/../../erwin/training.py", line 155, in benchmark_flops
    res = model.training_step(sample_batch)["train/loss"]
  File "/gpfs/home5/scur2687/erwin/experiments/../../erwin/experiments/wrappers/shapenet.py", line 46, in training_step
    return self.step(batch, "train")
  File "/gpfs/home5/scur2687/erwin/experiments/../../erwin/experiments/wrappers/shapenet.py", line 40, in step
    pred = self(**batch).squeeze(-1)
  File "/gpfs/home5/scur2687/erwin/erwin/lib64/python3.9/site-packages/torch/nn/modules/module.py", line 1736, in _wrapped_call_impl
    return self._call_impl(*args, **kwargs)
  File "/gpfs/home5/scur2687/erwin/erwin/lib64/python3.9/site-packages/torch/nn/modules/module.py", line 1844, in _call_impl
    return inner()
  File "/gpfs/home5/scur2687/erwin/erwin/lib64/python3.9/site-packages/torch/nn/modules/module.py", line 1790, in inner
    result = forward_call(*args, **kwargs)
  File "/gpfs/home5/scur2687/erwin/experiments/../../erwin/experiments/wrappers/shapenet.py", line 30, in forward
    node_features = self.pos_enc(node_positions)
  File "/gpfs/home5/scur2687/erwin/erwin/lib64/python3.9/site-packages/torch/nn/modules/module.py", line 1736, in _wrapped_call_impl
    return self._call_impl(*args, **kwargs)
  File "/gpfs/home5/scur2687/erwin/erwin/lib64/python3.9/site-packages/torch/nn/modules/module.py", line 1844, in _call_impl
    return inner()
  File "/gpfs/home5/scur2687/erwin/erwin/lib64/python3.9/site-packages/torch/nn/modules/module.py", line 1790, in inner
    result = forward_call(*args, **kwargs)
  File "/gpfs/home5/scur2687/erwin/experiments/../../erwin/experiments/wrappers/shapenet.py", line 12, in forward
    proj = self.linear(x)
  File "/gpfs/home5/scur2687/erwin/erwin/lib64/python3.9/site-packages/torch/nn/modules/module.py", line 1736, in _wrapped_call_impl
    return self._call_impl(*args, **kwargs)
  File "/gpfs/home5/scur2687/erwin/erwin/lib64/python3.9/site-packages/torch/nn/modules/module.py", line 1844, in _call_impl
    return inner()
  File "/gpfs/home5/scur2687/erwin/erwin/lib64/python3.9/site-packages/torch/nn/modules/module.py", line 1790, in inner
    result = forward_call(*args, **kwargs)
  File "/gpfs/home5/scur2687/erwin/erwin/lib64/python3.9/site-packages/torch/nn/modules/linear.py", line 125, in forward
    return F.linear(input, self.weight, self.bias)
  File "/gpfs/home5/scur2687/erwin/erwin/lib64/python3.9/site-packages/torchtnt/utils/flops.py", line 231, in __torch_dispatch__
    rs = func(*args, **kwargs)
  File "/gpfs/home5/scur2687/erwin/erwin/lib64/python3.9/site-packages/torch/_ops.py", line 716, in __call__
    return self._op(*args, **kwargs)
RuntimeError: Expected all tensors to be on the same device, but found at least two devices, cpu and cuda:0! (when checking argument for argument mat2 in method wrapper_CUDA_mm)
